### ISSUE TITLE
This IS still needed

### ISSUE THREAD

**Issue by DanielJMaxwell** [MichalBures/obsidian-file-path-to-uri#10]
Regarding your comment in the readme file that this plug-in isn't needed any more: the technique you describe does not work for folders, so your plug-in is in fact still needed.

**Comment by juanbretti** [comment]
Hey @DanielJMaxwell , check again. The folders now work with `Ctrl+drag` without this great plugin.

**Comment by DanielJMaxwell** [comment]
Great to hear, thank you for the update

**Comment by Karob2** [comment]
I still use this plugin because the Obsidian native way only displays the file/folder name, while I want to display the full path.

**Comment by generic-user** [comment]
Yes, I use this as well, this makes it possible to use keyboard to get the job done.
So please keep it updated, as it would be greatly missed! 👍 🎉
